license-tool.toml

```diff
@@ -0,0 +1,21 @@
+[overrides]
+"backon" = { origin = "https://github.com/Xuanwo/backon" }
+"bollard-stubs" = { origin = "https://github.com/fussybeaver/bollard" }
+"openssl-macros" = { origin = "https://github.com/sfackler/rust-openssl" }
+"serde_nanos" = { origin = "https://github.com/caspervonb/serde_nanos" }
+
+# These can go away once Vector starts using a release of the VRL crate with a
+# library field set up.
+"vrl" = { license = "MPL-2.0" }
+"vrl-compiler" = { license = "MPL-2.0" }
+"vrl-core" = { license = "MPL-2.0" }
+"vrl-diagnostic" = { license = "MPL-2.0" }
+"vrl-parser" = { license = "MPL-2.0" }
+"vrl-tests" = { license = "MPL-2.0" }
+
+"ring-0.16.20" = { license = "Custom" }
+"rustls-webpki-0.100.1" = { license = "ISC-style" }
+"webpki-0.21.4" = { license = "ISC-style" }
+"webpki-0.22.0" = { license = "ISC-style" }
+"zerocopy-0.6.1" = { license = "BSD-2-Clause" }
+"zerocopy-derive-0.3.2" = { license = "BSD-2-Clause" }
```

Comment:
Should each of these have a comment explaining?

Reply:
Oh, yeah, probably. Fixed in 4ed90410c